src/webgpu/api/validation/queue/writeBuffer.spec.ts

```diff
@@ -11,10 +11,115 @@ Tests writeBuffer validation.
 
 Note: destroyed buffer is tested in destroyed/.
 
-TODO: implement.
+TODO: implement usage flag validation.
```

Comment:
Another TODO I thought of while looking at copyBufferToBuffer: try to test huge values for the copy size that could overflow (e.g. if multiplied to bytes before comparing the size with the TypedArray) and end up being incorrectly treated as valid.

Reply:
Added the TODO